### PR TITLE
Test audio record

### LINKED ISSUE #206: Add tests for audio recording

### PR DESCRIPTION
**Pull request by yasmineben510**
Added tests for the audio recording, weren't pushed before by mistake

closes #206

### CODE REVIEW

File: app/src/androidTest/java/com/github/orkest/shazam/data/AudioRecordingTest.kt
Comment (saraanej) [inline app/src/androidTest/java/com/github/orkest/shazam/data/AudioRecordingTest.kt:1]: Good tests!

**Review by saraanej — APPROVED** [review]
Thank you for your work. The tests look good to me.

File: app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt [hunk 1]
```diff
@@ -112,6 +95,26 @@ class AudioRecording {
...
+//        fun logsRecordedAudio(coroutineScope: CoroutineScope){
+//            coroutineScope.launch {
+//                recordingFlow(coroutineScope).collect {
+//                    Log.d("HasOnesBuffer", it.buffer.sum().toString())
+//                    Log.d("LengthBuffer", it.buffer.size.toString())
+//                    Log.d("ShazamActivity", it.timestamp.toString())
+//                }
+//            }
+//
+//        }
```
[the remark this answers is not in the document]
Comment (yasmineben510) [reply app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt:115]: No not for the unit test but it’s useful for manual testing and debugging

[the remark this answers is not in the document]
Comment (yasmineben510) [reply app/src/main/java/com/github/orkest/shazam/data/AudioRecording.kt:115]: But yes I should remove it

**Pull request merged**